static/src/javascripts/projects/common/modules/experiments/tests/commercial-lazy-load-margin.ts

```diff
@@ -0,0 +1,30 @@
+import type { ABTest } from '@guardian/ab-core';
+import { noop } from '../../../../../lib/noop';
+
+export const commercialLazyLoadMargin: ABTest = {
+	id: 'CommercialLazyLoadMargin',
+	start: '2022-03-22',
+	// test should be in place for a minimum of 14 days
+	expiry: '2020-04-06',
```

Comment:
For the expiry, you might want to update this to give yourself another week or two in case it needs to run longer?

Reply:
Good call - I'll give it an extra week